src/views/design/ScriptDetail/index.tsx

```diff
@@ -332,9 +332,14 @@ const ScriptDetail = withStyles(styles)(
                                 label={translator('scripts.detail.side.script_description')}
                                 multiline
                                 rows={8}
-                                value={newScriptDetail && newScriptDetail.description
-                                    ? newScriptDetail.description : ''}
-                                onChange={(e) => this.updateScript({ description: e.target.value })}
+                                value={newScriptDetail && newScriptDetail.version.description
```

Comment:
please have a look at the CircleCI tests under the Checks tab. It seems that there are some eslint errors